label/reports/dynamic_label.xml

```diff
@@ -0,0 +1,80 @@
+<openerp>
+  <data>
+    <template id="report_label">
+      <style type="text/css">
```

Comment:
Please move this to `static/src/css/label.css` and add

`<link rel="stylesheet" href="/label/static/src/css/label.css"/>`

in `view/assets.xml`, see example [here](https://github.com/rz-comp/web/blob/8.0/web_tree_image/view/assets.xml)